extensions/utils/metricsUtil.js

```diff
@@ -0,0 +1,28 @@
+function getStatusLabel(err) {
```

Comment:
do we really need this?
"aggregating" by status code 'range' can be done at presentation time (in prometheus queries), and there are not that many code anyway ; yet it may useful at times to be able to dig and find the metric for a specific code ?

Reply:
Since lifecycle dashboards are already using the "aggregating by status code rage" logic, I see this more as a tech depth that will be tackled by https://scality.atlassian.net/browse/BB-342.

Reply:
TechDebt indeed, though very quick to fix : i.e. use the 'raw' status instead of calling `getStatusLabel` in the code, and replace each of these filters in dashboard (e.g. `="2xx"` with `=~"^2" for each of the ranges 2/3/4/5)